src/sentry/static/sentry/app/views/organizationDiscover/earlyAdopterMessage.jsx

```diff
@@ -1,20 +1,22 @@
 import React from 'react';
 import styled from 'react-emotion';
+import {Flex} from 'grid-emotion';
 
 import Alert from 'app/components/alert';
 import Link from 'app/components/link';
-import {tct} from 'app/locale';
+import {t} from 'app/locale';
 
 export default class EarlyAdopterMessage extends React.Component {
   render() {
     return (
-      <StyledAlert type="info" icon="icon-circle-exclamation">
-        {tct(
-          'This is an alpha release and may change in the future. Please email any feedback to [email:feedback-discover@sentry.io]. Thanks for being an early adopter!',
-          {
-            email: <Link href="mailto:feedback-discover@sentry.io" />,
-          }
-        )}
+      <StyledAlert type="info" icon="icon-labs">
+        <Flex justify="space-between">
+          {t('Sentry Discover is alpha software. Thanks for being an early adopter.')}
+          <Link href="mailto:feedback-discover@sentry.io">
+            {' '}
+            {t('feedback-discover@sentry.io')}{' '}
+          </Link>
```

Comment:
Probably because of alignment

Reply:
Billy helped me out with this decision: 
t = strings only
tct = placeholder replacement with a react element

I changed it to a string so I could separate them with space-between.

Reply:
We don't need `t()` here for an email address... you can't translate that.

Removed `tct` because of the sentence structure and you don't need the email placeholder to be able to translate that sentence.